plone/app/contenttypes/migration/vocabularies.py

```diff
@@ -55,49 +58,51 @@
     # Image without blobs
     "Image": {
         'iface': IATImage,
-        'migrator': migration.migrate_images,
+        'migrators': (migration.ImageMigrator,),
         'extended_fields': [],
         'type_name': 'Image',
         'old_meta_type': 'ATImage',
     },
     "News Item": {
         'iface': IATNewsItem,
-        'migrator': migration.migrate_newsitems,
+        'migrators': (migration.NewsItemMigrator,),
```

Comment:
Only because it wasn't already in that list and I didn't know whether there was a special reason why.

Philip Bauer <notifications@github.com> wrote:

>In plone/app/contenttypes/migration/vocabularies.py:
>
>> 'extended_fields': [], > 'type_name': 'Image', > 'old_meta_type': 'ATImage', > }, > "News Item": { > 'iface': IATNewsItem, > - 'migrator': migration.migrate_newsitems, > + 'migrators': (migration.NewsItemMigrator,), 
>
>@djowett why did you not add the BlobNewsItemMigrator as well?
>
>—
>Reply to this email directly or view it on GitHub.￼
>